### ISSUE TITLE
ClientOrderId

### ISSUE THREAD

**Issue by eav** [TinkoffCreditSystems/invest-openapi#505]
Было бы не плохо добавить так навываемый сlientOrderId (для сохранения метаданных) а также сделать его доступным в операциях.

**Comment by AlexanderVolkovTCS** [comment]
Да, у нас планируется ключ идемпотентности при выставлении заявок

**Comment by eav** [comment]
Есть примерно информация по срокам?

**Comment by AlexanderVolkovTCS** [comment]
Пока нет

**Comment by eav** [comment]
и на том спасибо )

**Comment by Dimantchick** [comment]
> Пока нет

А нет информации хотя бы о порядке сроков перехода на версию 2? Просто для понимания имеет ли смысл вести разработку под текущую версию, если при переходе все "сломается".

**Comment by leodev87** [comment]
Похоже мои внуки еще будут ждать вторую версию

**Comment by AlexanderVolkovTCS** [comment]
>А нет информации хотя бы о порядке сроков перехода на версию 2? 

В 2021 выпустим, обратной совместимости не будет ни по методам, ни по протоколу.

**Comment by kitMP** [comment]
А сколько при этом будет жить предыдущая версия?

**Comment by BaryshevS** [comment]
Ну да. Хотелось бы года 2-3, чтобы не отключали. Мали сколько еще в новой версии багов напридумываете. :)
Этих уже одомашнили.

**Comment by sergeymavrin** [comment]
да-да, уточните, пожалуйста, сколько будет текущая версия существовать после появления 2-ой версии? А то четверть кода точно работает на одомашнивание текущих багов, которые Вы не хотите править в текущей версии
